sourcegraph/sourcegraph
# [SG-29514] Favicon has fuzzy edges in Firefox running on macOS

## Description
<img width="131" alt="fuzzy" src="https://user-images.githubusercontent.com/7220167/158150237-dab3525d-c3a6-49b7-a3c6-d63ecb5d8a4d.png">

The edges of the capsules look fuzzy, especially for the sky blue capsule. This looks particularly odd next to other favicons which are often geometrically more complex.

## Success Criteria
- The favicon should look sharp/crisp, similar to how GitHub's favicon (shown on the right) looks crisp.

## Implementation Details
We should use an SVG for the favicon icon.
- I believe we currently use enterprise/ui/assets/img/favicon.png or ui/assets/img/favicon.png but we should instead use enterprise/ui/assets/img/sourcegraph-mark.svg or ui/assets/img/sourcegraph-mark.svg.

## Refs
- [Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/29514)
- [Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-29514)


## Test plan
- Open any page on sourcegraph.com.
- Run `sg start` or `sg start oss` to run the local instance.
- Compare the two favicon on Firefox macOs v12.
- The `local instance` should have crispy/sharp edges while the `remote instance` should still have fuzzy edges.
